### DISCUSSION TITLE
getDocTooltip Error

### DISCUSSION THREAD

**Discussion by ArunPatal** [ajaxorg/ace#5329] [Q&A]
Hello,
I am getting following error while displaying autoCompletion

`editor.completer.showPopup(editor);`

![bug](https://github.com/ajaxorg/ace/assets/30784805/8738dfbe-30ab-4072-a05a-c9e02e3b0606)

**Comment by nightwing** [comment]
Could you please show what code you are using to get this error.

 I could not reproduce it on ace-playground [with this](https://mkslanc.github.io/ace-playground/?value=CnZhciBlZGl0b3IgPSB3aW5kb3cuYWNlLmVkaXQoImVkaXRvci1kaXZJZCIsIHsKICAgIHZhbHVlOiBgZnVuIGhlbGxvOgogICAgcHJpbnQoJ3gnICsgMSkKICAgICMgZG9uZQpgLAoJbW9kZTogJ2FjZS9tb2RlL3B5dGhvbicsCgl0aGVtZTogJ2FjZS90aGVtZS9jcmltc29uX2VkaXRvcicsCglmb250U2l6ZTogMTUsCglzaG93R3V0dGVyOiB0cnVlLAoJc2hvd1ByaW50TWFyZ2luOiBmYWxzZSwKCWhpZ2hsaWdodEFjdGl2ZUxpbmU6IGZhbHNlLAoJdXNlV29ya2VyOiBmYWxzZSwKCWVuYWJsZUJhc2ljQXV0b2NvbXBsZXRpb246IHRydWUsCgllbmFibGVMaXZlQXV0b2NvbXBsZXRpb246IHRydWUsCn0pOwplZGl0b3IuZXhlY0NvbW1hbmQoInN0YXJ0QXV0b2NvbXBsZXRlIikKZWRpdG9yLm5hdmlnYXRlRmlsZUVuZCgpCmVkaXRvci5jb21wbGV0ZXIuc2hvd1BvcHVwKGVkaXRvcik7CgpcMFwwPHNjcmlwdCBzcmM9Imh0dHBzOi8vd3d3LnVucGtnLmNvbS9hY2UtYnVpbGRzQGxhdGVzdC9zcmMtbm9jb25mbGljdC9hY2UuanMiPjwvc2NyaXB0Pgo8c2NyaXB0IHNyYz0iaHR0cHM6Ly93d3cudW5wa2cuY29tL2FjZS1idWlsZHNAbGF0ZXN0L3NyYy1ub2NvbmZsaWN0L2V4dC1sYW5ndWFnZV90b29scy5qcyI%2BPC9zY3JpcHQ%2BCgo8ZGl2IGlkPSdlZGl0b3ItZGl2SWQnIHN0eWxlPSJoZWlnaHQ6IDEwMCUiPjwvZGl2Pgo%3D#hello-world)

**Reply by ArunPatal** [reply]
Hi,

Sorry, i just checked that i was setting => editor.completers[4] = My_Completer
but length if editor.completers is 2. After setting editor.completers[2] = My_Completer, it work :)

Setting wrong index throw this error

Thanks for quick review
